tests/tracer/test_tracer.py

```diff
@@ -1654,3 +1654,21 @@ def test_fork_manual_span_different_contexts(tracer):
     _, status = os.waitpid(pid, 0)
     exit_code = os.WEXITSTATUS(status)
     assert exit_code == 12
+
+
+def test_context_tags(tracer, test_spans):
```

Comment:
I can't really see how this added test relates to this PR 😕

Reply:
Yes - it's pretty unrelated. I'll do it as a separate change :+1: